primus-core/src/main/java/com/bytedance/primus/executor/worker/launchplugin/EnvPlugin.java

```diff
@@ -24,21 +24,16 @@
 import static com.bytedance.primus.apiserver.utils.Constants.PRIMUS_EXECUTOR_UNIQID_ENV;
 
 import com.bytedance.primus.common.child.ChildLaunchPlugin;
-import com.bytedance.primus.common.model.records.ContainerId;
 import com.bytedance.primus.executor.ExecutorContext;
 import com.bytedance.primus.executor.worker.WorkerContext;
-import com.bytedance.primus.proto.PrimusConfOuterClass.PrimusConf;
 import java.util.HashMap;
 import java.util.Map;
 import org.apache.commons.lang.StringUtils;
 
 public class EnvPlugin implements ChildLaunchPlugin {
 
-  private static final String APPLICATION_ID_ENV_KEY = "APPLICATION_ID";
-  private static final String CONTAINER_ID_ENV_KEY = "CONTAINER_ID";
   private static final String ORACLE_ID_ENV_KEY = "ORACLE_ID";
```

Comment:
Can we change the name of env key to EXECUTOR_ID

Reply:
Sorry, I don't quite understand, aren't they removed?

Reply:
ORACLE_ID is used by an internal component. For tf or pytorch, maybe we can use EXECUTOR_ID

Reply:
done :D